gems/awesome-bot/CVE-2019-15224.yml

```diff
@@ -0,0 +1,17 @@
+---
+gem: awesome-bot
+cve: 2019-15224
+ghsa: 333g-rpr4-7hxq
+url: https://github.com/rubygems/rubygems.org/issues/2097
+date: 2019-08-20
+title: Critical severity vulnerability that affects awesome-bot
```

Comment:
Can we get the titles updated to be specific to the vulnerability rather than these generic ones?

Reply:
Done.